docs/deploy.md

```diff
@@ -0,0 +1,119 @@
+# Deploying M3DB
+
+## Deploying a single node
+
+Deploying a single-node cluster is a great way to experiment with M3DB and get a feel for what it
+has to offer. Our Docker image by default configures a single M3DB instance with an embedded etcd
+server (used for runtime configuration flags and topology management). This will allow you to read
+and write timeseries data and change the topology of the instance.
+
+To begin, first start up a Docker container with port `7201` (used to manage the cluster topology)
+and port `9003` (used to read and write metrics) exposed. We recommend you create a persistent data
+directory on your host for durability:
+
+```
+docker run -p 7201:7201 -p 9003:9003 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db quay.io/m3db/m3db:latest
+```
+
+<!-- TODO: link to docs containing explanations of what namespaces, the coordinator,
+placements, etc. are -->
+
+<!-- TODO: add something about how this is in no way a recommended production deployment guide,
+and write a guide for what is considered a production-ready deployment (this is in the works) -->
+
+Next, create an initial namespace for your metrics:
+
+```
+curl -X POST localhost:7201/namespace/add -d '{
+  "name": "default",
+  "options": {
+    "bootstrapEnabled": true,
+    "flushEnabled": true,
+    "writesToCommitLog": true,
+    "enabledEnabled": true,
+    "retentionOptions": {
+      "retentionPeriodNanos": 172800000000000,
+      "blockSizeNanos": 7200000000000,
+      "bufferFutureNanos": 600000000000,
+      "bufferPastNanos": 600000000000,
+      "blockDataExpiry": true,
+      "blockDataExpiryAfterNotAccessPeriodNanos": 300000000000
```

Comment:
Side note: we should really allow "1m" and "1h" shorthands for this stuff in the future.

Reply:
Yeah that got changed when we moved to parsing the response directly into the `NamespaceOptions` proto message. We'll make this more ergonomic. Maybe we can use the protobuf [duration type](https://github.com/google/protobuf/blob/master/src/google/protobuf/duration.proto)?

It's a wrapper over nanos but might support go `ParseDuration`.